### ISSUE TITLE
Package request: torcheval

### ISSUE THREAD

**Issue by osbm** [NixOS/nixpkgs#376031]
### Nixpkgs version

- Previous Stable (24.05)

### Describe the package

 A library that contains a rich collection of performant PyTorch model metrics, a simple interface to create new metrics, a toolkit to facilitate metric computation in distributed training and tools for PyTorch model evaluations.

Its an an official pytorch package that is more or less is the replacement for the torchmetrics package from [lightning.ai](https://lightning.ai).

### Upstream homepage

https://pytorch.org/torcheval/stable/

### Source URL

https://github.com/pytorch/torcheval

### License

BSD License

### Platforms

x86_64-linux, aarch64-linux

### Additional context

This package is releasing versions but also states that its in its alpha phase. So I'd understand that if this makes this package undesirable for the nixpkgs until they start releasing stable versions. However some training scripts already make use of this package and i keep trying to implement the metric myself or make use of torchmetrics.

I can open a pull request but i guess it would be awesome to get greenlight (or redlight) before start working.

### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this package does not yet exist in an [open pull request](https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+is%3Apr+label%3A%228.has%3A+package+%28new%29%22) or in [Nixpkgs Unstable](https://search.nixos.org/packages?channel=unstable).
- [x] I assert that this is not a [duplicate of any known issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+label%3A%220.kind%3A+packaging+request%22).
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.

### For this package's maintainers:

Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

**Comment by osbm** [comment]
There must be a mistake. I think they forgot to push the tag to the github. I am going to ask them about it.

I think we should use the github version to run tests. we can update the package later when there is a proper tag update.

**Comment by osbm** [comment]
linking my issue: https://github.com/pytorch/torcheval/issues/215

**Comment by osbm** [comment]
closing as completed by #376129
